x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -84,6 +85,10 @@ const renderFormattedNumber = (value: number, _item: PolicyData) => (
   </TruncateText>
 );
 
+const renderPolicyNameLink = (value: string, _item: PolicyData) => {
+  return <Link to={`/policy/${_item.id}`}>{value + ' '}</Link>;
```

Comment:
Should use `EuiLink`. Can use the `onClick` event along with `react-router-dom` `useHistory()` to to push the route and then `ev.preventDefault()`. 
From your last PR, @oatkiller had suggested a generic method/`hook?` that we could add to our `/lib/` and then re-use with Eui Components that act like navigation links. Maybe its time to add it to our code base.

Reply:
@paul-tavares using EuiLink now.  I tried implementing the hook, but haven't quite gotten it to work yet.  I created this ticket last week: https://github.com/elastic/endpoint-app-team/issues/230

I can take that on next and isolate the effort and update all Links.  Are you OK with that?